iris/__init__.py

```diff
@@ -279,24 +255,19 @@ def register_extensions(app):
     app.register_blueprint(user_app, url_prefix="/user")
 
 # Decide whether to parse CLI args. If help is requested or the first token
-# looks like an IRIS mode ('demo'/'label'), let argparse handle args. This
-# allows `iris --help` to show argparse help while still avoiding accidental
-# parsing of pytest flags.
-if _cli_should_parse(sys.argv[1:]):
-    # Pass the CLI tokens through so argparse sees '-h'/'--help' when present.
-    args = parse_cmd_line(sys.argv[1:])
-else:
```

Comment:
I think this odd logic was from getting existing commands, `-- [non-iris-args]`, and `--help` to work simultaneously. thanks for cleaning it up.